be/src/column/fixed_length_column_base.cpp

```diff
@@ -319,7 +319,7 @@ std::string FixedLengthColumnBase<T>::debug_string() const {
     std::stringstream ss;
     ss << "[";
     size_t size = this->size();
-    for (size_t i = 0; i < size - 1; ++i) {
+    for (size_t i = 0; i + 1 < size; ++i) {
```

Comment:
why not replace with
```
for (int i - 0; i < size - 1; ++i) {
```

Reply:
you can run the code, which is a loop as size -1 overflows.
```
#include<iostream>  
#include<vector>
using namespace std;  

int main()  
{  
   vector<int> vint;
   for (int i = 0; i < vint.size() - 1; ++i) {
       cout << (vint.size()-1) << endl;
   }
   cout<< "endl" << endl;
}
```

Reply:
how about change the code to
```
int64_t size = this->size();
```

Reply:
i think the PR is better to avoid implicit cast.